src/hotel_reservation/paths.py

```diff
@@ -0,0 +1,46 @@
+import os
+from pathlib import Path
+
+import yaml
+from dotenv import load_dotenv
+
+
+class AllPaths:
+    def __init__(self) -> None:
+        load_dotenv()  # Load environment variables from .env file if present
+
+        if self.is_databricks():
+            # Running in Databricks
+            self.filename_config = Path(
+                os.environ.get(
+                    "DATABRICKS_CONFIG_PATH",
+                    "/Workspace/Users/cremerfederico29@gmail.com/marvelmlops-cremerf/project-config.yml",
+                )
+            )
+        else:
+            # Running in local environment
+            self.BASE_DIR = Path(__file__).resolve().parent.parent.parent
+            self.filename_config = self.BASE_DIR / "project-config.yml"
+
+        self.config = self.get_config_file()
+        self.cremerf_catalog = f'/Volumes/{self.config["catalog_name"]}/{self.config["schema_name"]}/'
```

Comment:
I would avoid adding your name into attribute names

Reply:
i'll change in the next submission